tests/phpunit/tests/editor/navigation-fallback.php

```diff
@@ -0,0 +1,338 @@
+<?php
+/**
+ * Tests WP_Navigation_Fallback
+ *
+ * @package WordPress
+ */
+
+/**
+ * Tests for the WP_Navigation_Fallback class.
+ */
+class WP_Navigation_Fallback_Test extends WP_UnitTestCase {
+
+	protected static $admin_user;
+	protected static $editor_user;
+
+	public static function wpSetUpBeforeClass( $factory ) {
+		self::$admin_user = $factory->user->create( array( 'role' => 'administrator' ) );
+
+		self::$editor_user = $factory->user->create( array( 'role' => 'editor' ) );
+	}
+
+	public function set_up() {
+		parent::set_up();
+
+		wp_set_current_user( self::$admin_user );
+	}
+
+	/**
+	 * @covers WP_REST_Navigation_Fallback_Controller
+	 */
+	public function test_it_exists() {
+		$this->assertTrue( class_exists( 'WP_Navigation_Fallback' ), 'WP_Navigation_Fallback class should exist.' );
+	}
+
+
+	/**
+	 * @covers WP_REST_Navigation_Fallback_Controller::get_fallback
```

Comment:
Unit tests need a ticket number of them.

Reply:
> ...a ticket number of them

I'm not sure what a "ticket number" is. Can you advise further?

Reply:
@getdave https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#annotations

Example - https://github.com/WordPress/wordpress-develop/commit/1a5b52a17e0bfa1ca2d9133466b5ead098e3a1ba#diff-5c695d8428b1720cb6a9ba1d9f7c8cc5365125084dbd20887b228f937a2580b6R290

Reply:
Thank you for the context. 

> Use @ticket 12345 to indicate that a test addresses the bug described in ticket...

I read that and I was slightly confused because these files do not fix a bug. However, I assume this is some convention by way that when a backport is introduced we tag the Trac ticket to the test where it was introduced?

If I've got that wrong please do let me know.